roles/openshift_metrics/tasks/start_metrics.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- command: >
+    {{openshift.common.client_binary}} 
+    --config={{mktemp.stdout}}/admin.kubeconfig 
+    get rc 
+    -l metrics-infra=hawkular-cassandra 
+    -o name 
+    -n {{openshift_metrics_project}}
+  register: metrics_cassandra_rc
+  changed_when: no
+
+- name: Start Hawkular Cassandra
+  include: scale.yaml
+  vars:
+    desired: 1
```

Comment:
should this be hardcoded to 1 here? Are we expecting petsets to be usable in the future for Cassandra?